lib/3scale/backend/stats/cleaner.rb

```diff
@@ -245,6 +252,16 @@ def delete_stats_keys_with_val_0(redis_conn, log_deleted_keys)
               sleep(SLEEP_BETWEEN_SCANS)
             end
           end
+
+          def with_retries
```

Comment:
Not important, but would be nice for this or other more important places: `def with_retries(max = MAX_RETRIES_REDIS_ERRORS)`

Reply:
Fair enough. Changed :+1: